scaladoc/src/dotty/tools/scaladoc/renderers/MemberRenderer.scala

```diff
@@ -202,22 +202,34 @@ class MemberRenderer(signatureRenderer: SignatureRenderer)(using DocContext) ext
     case m: Member => m.inheritedFrom.nonEmpty
     case g: MGroup => g.members.exists(isInherited)
 
+  private def isAbstract(m: Member | MGroup): Boolean = m match
+    case m: Member => m.modifiers.contains(Modifier.Abstract)
+    case g: MGroup => g.members.exists(isAbstract)
```

Comment:
Will it mark object as abstract if it contains an abstract class? e.g.:
```object A { abstract class B  { abstract def foo: Int } }```

Reply:
I added tests to cover that case